### DISCUSSION TITLE
Getting policy violation errors on UPDATE now on only 'deleted_at' fields for various tables

### DISCUSSION THREAD

**Discussion by stefanteulon** [supabase/supabase#26677] [Questions]
I'm getting a policy violation error when trying to update my "deleted_at" field in my table.

ERROR: 42501: new row violates row-level security policy for table "recipes"

This ONLY applies to that field, all other fields (including other timestampz fields) appear to successfully update with the same policy.

I'm also getting the same issue with all my tables that have a deleted_at field in them now, what's going on? Has 'deleted_at' become a special field title in supaBase?

This behavior is new, like within the last 24 hours.

Table: recipes

Name	Format	Type	Description
id	uuid	string	
user_id	uuid	string	
title	character varying	string	
deleted_at	timestamp with time zone	string	
Policy

alter policy "update_own_recipes"
on "public"."recipes"
to public
using (
  (auth.uid() = user_id)
);

**Comment by GaryAustin1** [comment]
That is all Postgres, so delete_at should not have changed "meaning".
 Do you have a trigger function on update?
 Do you have ANY policies that use deleted_at, like a select policy to not show the row if set?
